src/assets/js/github.js

```diff
@@ -53,7 +53,7 @@ function redirectIfQuery() {
     var pageName = window.location.toString(),
         delimiterPosition,
         redirectURL = 'https://github.com/Esri?q=',
-        topicList = document.getElementById('esri-topic-list').childNodes
+        topicList = document.getElementById('esri-topic-list').childNodes,
```

Comment:
it seemed to me that we were redirecting with no query even if we received no query, so I changed it to only check if a query was requested